Core/src/org/sleuthkit/autopsy/healthmonitor/EnterpriseHealthMonitor.java

```diff
@@ -276,12 +282,34 @@ public static void submitTimingMetric(TimingMetric metric) {
         }
     }
     
+    /**
+     * Submit the metric that was previously obtained through getTimingMetric(),
+     * incorporating a count that the time should be divided by.
+     * Call this immediately after the section of code being timed.
+     * This method is safe to call regardless of whether the Enterprise Health
+     * Monitor is enabled.
+     * @param metric The TimingMetric object obtained from getTimingMetric()
+     * @param normalization The number to divide the time by (a zero here will be treated as a one)
+     */
+    public static void submitNormalizedTimingMetric(TimingMetric metric, long normalization) {
```

Comment:
I think it would be nice to standardize on either "submit" or "add", e.g., submitTimingMetric and submitNormalizedTimingMetric.

Reply:
There is a standard - "submit" is what clients do, those are our public methods. They're submitting a metric to the health monitor, and the metric might get ignored if the monitor isn't enabled. "add" is when the monitor adds the metric to its internal list.